manager/integration/tests/test_settings.py

```diff
@@ -642,7 +646,9 @@ def test_setting_backing_image_auto_cleanup():  # NOQA
         4. Verify the only backing image file in each disk is reused by
            multiple replicas. The backing image file path is
            `<Data path>/<The backing image name>/backing`
-    5. Decrease the replica count by 1 for all volumes.
+    5. Enable 'Replica Node Level Soft Anti-Affinity' and unschedule test node
+       to guarantee if replica removed from test node, no new replica will
+       generte on it.
```

Comment:
` no new replica will generte on it` -> `no new replica can be rebuilt on the test node`

Reply:
I midified the code